src/object/explosion.cpp

```diff
@@ -76,7 +76,7 @@ Explosion::explode()
 
   if (push) {
     Vector center = m_col.m_bbox.get_middle ();
-    auto near_objects = Sector::get().get_nearby_objects (center, 10.0 * 32.0);
+    auto near_objects = Sector::get().get_nearby_objects (center, 128.0 * 128.0);
```

Comment:
I think the 32.0 factor is there to have the value in tiles, so 10.0 * 32.0 means 10.0 tiles.
Here the argument corresponds to 512 tiles: 128.0*128.0=512.0*32.0

Reply:
I might change/fix that, but bigger number = more significant explosion :P